packages/clarity-js/src/core/scrub.ts

```diff
@@ -64,11 +84,8 @@ function mask(value: string): string {
 
 function mangleToken(value: string): string {
     let length = ((Math.floor(value.length / Data.Setting.WordLength) + 1) * Data.Setting.WordLength);
-    let output: string = Layout.Constant.Empty;
-    for (let i = 0; i < length; i++) {
-        output += i > 0 && i % Data.Setting.WordLength === 0 ? Data.Constant.Space : Data.Constant.Mask;
-    }
-    return output;
+    let checksum = value.length >= Data.Setting.WordLength && config.fraud ? hash(value, Data.Setting.ChecksumPrecision) : Data.Constant.Zero;
```

Comment:
Use a value that is not a legal hash like -1 or empty.

Reply:
Went with empty.